politeiawww/cmd/politeiawwwcli/commands/censorcomment.go

```diff
@@ -8,6 +8,28 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+// Help message displayed for the command 'politeiawwwcli help censorcomment'
+var CensorCommentCmdHelpMsg = `censorcomment "token" "commentID" "reason"
+
+Censor comment (admin).
+
+Arguments:
+1. token       (string, required)   Proposal censorship token
+2. commentID   (string, required)   Id of the comment
+3. reason      (string, required)   Reason for censoring the comment
+
+Result:
```

Comment:
The first object that you have listed under `Result` is not actually part of the result.  It is the request body that is being sent up to the server.  Most of the time this is going to be the same as `Arguments`, unless a user signature.  I would do something like this:

```
censorcomment "token" "commentID" "reason"

Censor a user comment.  Requires admin privileges.

Arguments:
1. token       (string, required)   Proposal censorship token
2. commentID   (string, required)   Id of the comment
3. reason      (string, required)   Reason for censoring the comment

Request:
{
  "token":      (string)  Censorship token
  "commentid":  (string)  Id of comment
  "reason":     (string)  Reason for censoring the comment
  "signature":  (string)  Signature of censor comment (Token+CommentID+Reason)
  "publickey":  (string)  Public key used for signature
}

Response:
{
  "receipt":  (string)  Server signature of comment sensor signature
}
```

Reply:
Have applied the `Request:`, `Response:` format to all commands that output the request:

censorcomment
changepassword
changeusername
manageuser
edituser
Newcomment
newuser
setproposalstatus
votecomment